include/librealsense2/h/rs_types.h

```diff
@@ -222,6 +222,7 @@ typedef enum rs2_extension
     RS2_EXTENSION_DEBUG_STREAM_SENSOR,
     RS2_EXTENSION_CALIBRATION_CHANGE_DEVICE,
     RS2_EXTENSION_SAFETY_SENSOR,
+    RS2_EXTENSION_MAPPING_SENSOR,
```

Comment:
Also spoke about this with Evgeni, how about naming the new sensor "3D_MAPPING_SENSOR"?

Reply:
As we spoke, it will be depth_mapping_sensor